src/components/wizard-navigation-bar.component.horizontal.less

```diff
@@ -46,33 +46,62 @@ aw-wizard-navigation-bar.horizontal {
     left: calc(50% ~'-' @dot-width / 2);
     width: @dot-width;
     height: @dot-height;
-    content: '';
     text-align: center;
     vertical-align: middle;
     line-height: @dot-height - 2 * @dot-border-width;
     transition: 0.25s;
     border-radius: 100%;
   }
+  .state-circle-hover(@dot-width, @dot-height, @dot-border-width) {
+  }
+  .state-circle-nohover(@dot-width, @dot-height, @dot-border-width) {
```

Comment:
These two class definitions looks strange to me, because they are empty.
Are they required?

Reply:
I have made them for consistency with definitions of other states. In my opinion,  it is nice to always have

```
.state-zzz { ... }
.state-zzz-hover { ... }
.state-zzz-nohover { ... }
```
and apply these mixins universally, even though some of these definitions happen to be empty at the moment. Another benefit is that it will be possible to quickly update these mixins in case some styling for hovered state is needed. And empty mixins have no runtime cost because they are optimized away during build.

Reply:
Yes, you're right about that.